README.md

```diff
@@ -1,26 +1,270 @@
+such as how the drive and corner commands are being calculated
+* The [Arduino readme](Arduino/README.md) details the code that runs on the Arduino, used to control the LED screen.
+
+## 1. Flashing the Arduino code
+
+In this section we will be flashing the code that runs on the arduino to control the LED matrix in the head. The following steps should be performed on your laptop or development machine (not the raspberry pi)
+
+1. Install the [Arduino IDE](https://www.arduino.cc/en/Main/Software) used for loading code onto the arduino
+2. Download the Arduino code:
+    1. Navigate to [the repo](https://github.com/nasa-jpl/osr-rover-code) and click the green "Clone or download"
+button. Choose "Download ZIP".
+    2. Unzip/extract and open the downloaded zip file. Then, select the Arduino folder and create a
+new zip file of just that Arduino folder. Name it OsrScreen.zip
+3. Load the sketch onto the Arduino
+    1. Unplug the Arduino shield JST cable so the Arduino isn’t powered by the control board
+    2. Connect the Arduino to your development machine with USB cable
+    3. Open Arduino IDE
+    4. Select Sketch -> Include Library -> Add .Zip Library
+    5. Select the OsrScreen.zip folder created previously
+    6. Click the Upload button in the Sketch Window
+4. To load the example in the Arduino IDE: File -> Examples -> OsrScreen -> OsrScreen
+
+## 2. Setting up the Raspberry Pi (RPi)
+
+In this section, we’ll go over setting up the Raspberry Pi (RPi) and setting up all the code that will run the rover. Our rover uses ROS (Robotic Operating System), which will be installed on your choice of operating system; we will set these up below.
+
+These instructions should work for both the RPi 3 and 4. You are free to use other versions of RPi, ROS, or OS, but setting these up is not covered here and it is not guaranteed that those will work.
+
+### 2.1 Installing the operating system
+
+The first step is to install the Ubuntu (Recommended), or Raspbian OS on your Raspberry Pi.
+
+#### 2.1.1 Ubuntu 18.04 (recommended)
+
+Download Ubuntu 18.04 from [here](https://ubuntu.com/download/raspberry-pi) for your RPi version. Go for the 64 bit version. **Note that as of 7/30/2020, the link for 18.04 64bit for rpi4 is broken, and points to the rpi3 version. You can grab the correct rpi4 version from [this direct link](https://ubuntu.com/download/raspberry-pi/thank-you?version=18.04&versionPatch=.4&architecture=arm64+raspi4)**.
+
+Follow the instructions on the download page for preparing the image for the RPi. Namely:
+
+- Flash Ubuntu onto your microSD card. There are instructions for doing this on [Ubuntu](https://ubuntu.com/tutorials/create-an-ubuntu-image-for-a-raspberry-pi-on-ubuntu#1-overview), [Windows](https://ubuntu.com/tutorials/create-an-ubuntu-image-for-a-raspberry-pi-on-windows), and [Mac](https://ubuntu.com/tutorials/create-an-ubuntu-image-for-a-raspberry-pi-on-macos)
+- Attach a monitor and keyboard to the RPi
```

Comment:
protip: you can actually get around this by using `screen` and USB-TTL: https://elinux.org/RPi_Serial_Connection

Reply:
Yeah. Could add a variant in there.

Reply:
Added a tiny note on it.